src/Model/Behavior/UploadBehavior.php

```diff
@@ -149,14 +147,8 @@ public function afterDelete(Event $event, Entity $entity, ArrayObject $options)
             }
 
             $writer = $this->getWriter($entity, [], $field, $settings);
-            $success = $writer->delete($files);
-
-            if ($result && (new Collection($success))->contains(false)) {
-                $result = false;
-            }
+            $writer->delete($files);
         }
-
-        return $result;
```

Comment:
Cake 3 doesn't check the return value of `afterDelete` callback, so this was pointless.